plugins/home/src/templates/TemplateBackstageLogo.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+type Classes = {
+  svg: string;
+  path: string;
+}
+
+const TemplateBackstageLogo = ({ classes }: { classes: Classes }) => {
```

Comment:
@backstage/reviewers I copied the Logo and LogoIcon from the Root now, as I had to use them in the template (and make some adjustments such as this one to accept classes prop). But if you think we should make them available from a core package, let me know!